build.sbt

```diff
@@ -269,6 +269,26 @@ lazy val root = (project in file("."))
     createTarballSettings()
   )
 
+lazy val spark = (project in file("connectors/spark"))
+  .dependsOn(client % "compile->compile;test->test")
+  .dependsOn(server % "test->test")
+  .settings(
+    name := s"$artifactNamePrefix-spark",
+    scalaVersion := "2.13.14",
+    commonSettings,
+    javaOptions ++= Seq(
+      "--add-opens=java.base/sun.nio.ch=ALL-UNNAMED",
```

Comment:
what is this for?

Reply:
Spark accesses some private JDK APIs, which is not allowed in Java 11+. Need special JVM options to make it work.

Reply:
Spark project does it as well.